packages/preview/src/browser/markdown/markdown-preview-handler.spec.ts

```diff
@@ -3,68 +3,54 @@
  *
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * @jest-environment @theia/core/src/browser/test/jsdom-environment
  */
 
-import { enableJSDOM } from '@theia/core/lib/browser/test/jsdom';
-
-let disableJSDOM = enableJSDOM();
```

Comment:
@ishitatsuyuki, I remember now, why it looks so complicated with `enableJSDOM`. this line is a good example. If you run that `markdown-preview-handler.spec.ts` isolated, so when there was no jsdom configures before, it will load modules which rely on some parts of dom. 
can you verify, that this test can be executed isolated?

Reply:
I'm not sure I understand. What do you mean by "isolated"?

Reply:
Jest uses sandboxes and spawns worker processes for each test suite by default with the exception of CI. Even if you don't disable something, things are isolated by default now so mistakes like this can be detected, if I understood your question correctly.

Reply:
"Run Mocha Test" launch config for instance. it won't load other modules that might configure jsdom beforehand, and that means loading of `markdown-preview-handler.spec.ts` itself would fail at `import`(actually `require`) if DOM is required, but not available.

Reply:
The motivation was to have the same condition for test execution for single execution as well as for all tests.

Reply:
>  Jest uses sandboxes and spawns worker processes for each test suite

Good to know. 

I'll try that out, but actually there is a test (`logger.spec.ts`) which should fail, if jsdom would be left over by previously loaded modules.